website/docs/advanced-concepts/granular-access-control/reference/permissions.mdx

```diff
@@ -0,0 +1,717 @@
+---
+description: Permissions in Granular Access Control in Appsmith
```

Comment:
We should add modules permissions to all this.

Reply:
Once we have modules permission - this will be taken care as part of a different ticket.